scripts/create-fuels-template-integration.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+(cd templates/nextjs && pnpm run fuels:dev) &
+cp templates/nextjs/.env.example templates/nextjs/.env.local
+(cd templates/nextjs && pnpm run dev) &
+pnpm exec playwright install --with-deps
+pnpm exec playwright test
```

Comment:
I may be a bit out of context, so here's a couple questions:
 1. Have you successfully run this on CI?
 1. Should we be concerned about shutting down the `dev` processes to avoid blocking?
 1. Should we run the validations using the published version from NPM? (cc @petertonysmith94)
    - Example: [packages/create-fuels/test/e2e.test.ts](https://github.com/FuelLabs/fuels-ts/pull/2364/files#diff-adad36ad07b62532f988d67772fa9eac2fa210d765683f38b6a78c6d6965ab9a)

Am I missing something?

Reply:
Merging this for now. If anything is required, we'll take care of it in #2170